.java-version

```diff
@@ -0,0 +1 @@
+temurin64-17.0.8.1
```

Comment:
I assume this is accidental addition?

Reply:
Right 🥲
It's from `jenv` I used to manage local, per-project java versions.
THankss